src/test/resources/com/puppycrawl/tools/checkstyle/utils/checkutils/InputCheckUtilsTest.java

```diff
@@ -0,0 +1,69 @@
+import java.util.HashMap;
```

Comment:
Input files should have package and they should be correct unless it is for testing something specific.
Eclipse gives me errors on this because we set inputs to be compilable.